### ISSUE TITLE
Build merge-to-main-cart on avocano-cart: FAILURE (Sold out!)

### ISSUE THREAD

**Issue by glasnt** [GoogleCloudPlatform/avocano#186]
Cloud Build encountered an error: 

- Project: avocano-cart
- Trigger: merge-to-main-cart
- Commit: [52ebacf](https://github.com/glasnt/avocano/commit/52ebacf61685bcc67f83834fbdb07f5507d7e3b1) on branch main

[View Logs](https://console.cloud.google.com/cloud-build/builds/d376c9c3-fc50-43b2-a185-ff784510b064?project=751300358416&utm_campaign=google-cloud-build-notifiers&utm_medium=http&utm_source=google-cloud-build)

<sub>I'm an automation.  Googlers: go/avocano-admin-notifiers</sub>

**Comment by glasnt** [comment]
Would you believe, not a network timeout!

**Comment by glasnt** [comment]
```
    def get_inventory():
        inventory = page.locator(".inventory").inner_text()
>       assert "Only" in inventory
E       AssertionError: assert 'Only' in 'Sold Out!'
```

Avocados sold out! Bumped the inventory. 

Might be an interesting -- but very low priority -- feature request to add alerting when products sell out.

**Comment by glasnt** [comment]
This recently appeared on avocano buynow because a misconfiguration meant the UI tests weren't being run, they're running now, but the product is sold out. 

Short-term fix was to manually add 999 instances of the flagship product

**Comment by glasnt** [comment]
Being addressed in #327
